docs/blog_post.md

```diff
@@ -0,0 +1,56 @@
+DRAFT ONLY--DO NOT QUOTE
+
+# ShopRunner Data Science for Good: Automating Urban Wildlife Classification
+
+![coyote](https://github.com/uptake/autofocus/raw/master/gallery/coyote1.jpg) 
+<!--TK image caption formatting-->
+Software developed by data scientists at ShopRunner and Uptake supports the work of urban wildlife researchers by automatically classifying images taken by motion-activated cameras, such as this image of a coyote in the Chicago area.
+
+The growth of cities threatens animal species that do not adapt well to urban environments. **The Lincoln Park Zoo's Urban Wildlife Institute studies how urbanization affects wildlife in part by collecting images from motion-activated cameras**, beginning in 2010 around Chicago but now from more than twenty urban areas. For almost a decade, volunteers have been helping in their efforts by identifying the animal species that appear in each of those images in the online [Zooniverse](https://www.zooniverse.org/projects/zooniverse/chicago-wildlife-watch) crowdsourcing platform. However, the rate of image collection now exceeds the rate of image labeling. The information that the images contain is effectively unavailable to researchers until they have been labeled, so **image labeling has become a bottleneck that holds back their research.**
+
+Fortunately, it is now possible to create computer programs that label images with accuracy comparable to that of humans. Moreover, ShopRunner's data science team has experience building such programs to classify products across ShopRunner's retail network for its marketplace app District. For over two years, **data scientists from ShopRunner and Uptake have been working with the Urban Wildlife Institute to use automated image classification techniques to unblock the institute's research efforts.** 
+
+**The data and code used for this project are [publicly available](https://github.com/uptake/autofocus) so that anyone interested in developing the relevant skills can get involved.**
+
+## Impact
+
+> It was like they gifted us 10 interns who could work around the clock. Autofocus enables us to get on top of our backlog of images and deliver valuable insights on local wildlife now.
+>
+> --Urban Wildlife Institute researcher Mason Fidino, PhD <!--TK image of Mason-->
+
+Approximately 10-15 percent of the images that the Urban Wildlife Institute's cameras capture contain humans rather than the wildlife that is relevant to the institute's research. Those images are a major nuisance because the institute wishes to respect privacy by filtering out those images before posting the rest online for volunteers to label.
+
+![](human_example.JPG)
+<!--TK image caption formatting-->
+The Urban Wildlife Institute's cameras often capture images of people jogging or walking dogs, or (as in this case) images of the researchers themselves stopping by to service them.
```

Comment:
...stopping by to service a deployed camera (using 'them' creates some confusion as what what we are trying to service.)